app/assets/stylesheets/ama_layout/layout_components/button-grouping.scss

```diff
@@ -35,7 +35,6 @@
 
   &__button{
     @extend .button;
-    margin-bottom: 0; //assuming the button lives within a container, which causes 2 sets of margins
```

Comment:
the issue was that before the margin was being applied twice, so now it only gets applied once, so no override is required